packages/openapi3/test/metadata.test.ts

```diff
@@ -517,6 +517,7 @@ describe("openapi3: metadata", () => {
             },
           },
           requestBody: {
+            description: "The body.",
```

Comment:
You might need to send a comparable PR on the `typespec-azure` side too because `Body<T>` may be used in Azure Core

Reply:
https://github.com/Azure/typespec-azure/pull/3477